### PR TITLE
User, access token and API key permissions

### PR DESCRIPTION
**Pull request by faissalMT**
What - Restricts user, access token and api key access to only specified projects

### CODE REVIEW

**Comment by jpmaHub** [comment]
It advised to write your commit messages in the imperative, so that you’re following the Git’s own built-in convention. Also, It would be better to squash or rebase your commits because some of the commits are failing.

File: spec/acceptance/local_authority/authorise_user_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require 'rspec'
 require_relative '../shared_context/use_case_factory'
 
-describe 'Authorises the user' do
+xdescribe 'Authorises the user' do
```
Comment (robinlacey) [inline spec/acceptance/local_authority/authorise_user_spec.rb]: assume this is still wip?

Comment (faissalMT) [reply spec/acceptance/local_authority/authorise_user_spec.rb]: No this needs to be removed

Comment (faissalMT) [reply spec/acceptance/local_authority/authorise_user_spec.rb]: No it needs to be removed.

**Pull request merged**